grid2op/multi_agent/utils.py

```diff
@@ -74,4 +77,74 @@ def __eq__(self, other):
             and self._current_agent == other._current_agent
             and self.selected_agent == other.selected_agent
         )
-        
\ No newline at end of file
+
+
+class ClusterUtils:
```

Comment:
Can you instead make a module (folder) names cluster_utils or clusterutils
And have this class named eg "LouvainClustering" to be more explicit 😊

Reply:
Hi Benjamin @BDonnot . Addressed this comment by moving the ClusterUtils in to a seperate module and renamed it to LouvainClutering

Reply:
Added unit testing for the **create_connectivity_matrix** and **cluster_substations** methods and added author names in AUTHORS.txt